plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/internal/db2/DB2GeoResource.java

```diff
@@ -199,7 +199,7 @@ protected DB2GeoResourceInfo createInfo(IProgressMonitor monitor) throws IOExcep
 
             icon = Glyph.icon(ft);
             keywords = new String[] { "db2", //$NON-NLS-1$
-                    ft.getName().getLocalPart(), ft.getName().getNamespaceURI() };
+                    ft.getName().getLocalPart(), ft.getName().getURI() };
```

Comment:
should be similar to shp implementation. check for namespace, if not null, add it otherwise skip it